tests/suites/test_suite_pkcs11_client.function

```diff
@@ -355,3 +355,194 @@ exit:
     mbedtls_pk_free( &transparent_ctx );
 }
 /* END_CASE */
+
+/* BEGIN_CASE depends_on:MBEDTLS_PK_C:MBEDTLS_SHA256_C */
+void pk_signX_verifyX( int key_type )
```

Comment:
We don't use capital letters in function names, and this name does not match what the function does (`X` looks like some kind of variable). The function needs to indicate what is tested, but generally not the other steps that enable the test. So how about `pk_generate_sign_verify` for this function, `pk_verify_signed` for the function that calls Cryptoki to verify a signature generated by the test, and `pk_verify` for a function that verifies a signature that's hard-coded in the test data?